includes/api/class-edd-api.php

```diff
@@ -92,7 +92,43 @@ class EDD_API {
 	 * @access private
 	 * @since 1.7
 	 */
-	private $override = true;
+	public $override = true;
+
+	/**
+	 * Version of the API queried
+	 *
+	 * @var string
+	 * @access public
+	 * @since 2.4
+	 */
+	private $queried_version;
+
+	/**
+	 * All versions of the API
+	 *
+	 * @var string
+	 * @access public
+	 * @since 2.4
+	 */
+	protected $versions = array();
```

Comment:
if this is gonna be protected, there needs to be a getter for this. Some extensions might want to use this